sehee/models/postsDao.js

```diff
@@ -0,0 +1,126 @@
+const { appDataSource } = require("./appDataSource");
+
+//게시글 등록
+const createPost = async (title, content, post_image, user_id) => {
+  try {
+    const data = await appDataSource.query(
+      `INSERT INTO posts(
+      title,
+      content,
+      post_image,
+      user_id
+    ) VALUES (?, ?, ?, ?);
+    `,
+      [title, content, post_image, user_id]
+    );
+    return data;
+  } catch (err) {
+    const data = new Error("DATA_IS_NOT_VAILD");
+    err.statusCode = 500;
+    throw err;
+  }
+};
+
+// 게시글 조회
+const getPosts = async () => {
+  try {
+    const data = await appDataSource.query(
+      `SELECT
+    u.id AS usersId,
+    u.profileImage AS userProfileImage,
+    p.user_id AS posingId,
+    p.post_image AS postingImageUrl,
+    p.content AS postingContent
+    FROM posts p INNER JOIN users u ON p.user_id = u.id;
+    `
+    );
+    return data;
+  } catch (err) {
+    const data = new Error("DATA_IS_NOT_VAILD");
+    err.statusCode = 500;
+    throw err;
+  }
+};
+
+// 유저 게시글 조회
+const getPostByUserId = async (userId) => {
+  try {
+    const data = await appDataSource.query(
+      `SELECT
+        u.id AS userId,
+        u.profileImage AS userProfileImage,
+        JSON_ARRAYAGG(
+        JSON_OBJECT('postingId', p.id,
+        'postingImageUrl', p.post_image, 'postingContent', p.content))
+          AS postings
```

Comment:
들여쓰기과 적절한 line-breaking을 적용해주세요
```suggestion
        JSON_ARRAYAGG(
          JSON_OBJECT(
            'postingId', p.id,
            'postingImageUrl',
            p.post_image,
            'postingContent', p.content
          )
        ) AS postings
```